src/services/index.js

```diff
@@ -1,7 +1,8 @@
 import axios from "axios";
+import env from "../../env.json";
 
 const orgApi = axios.create({
-  baseURL: "http://localhost:9999/api/"
+  baseURL: env.SERVICE_API_URL
```

Comment:
How does this end up selecting the correct environment?

Reply:
"env" isn't actually pulling from NODE_ENV, unfortunately-- there are a few ways to do that on React Native, but I tried most of them and they are either very complicated, broken, or no longer being supported. I made an env.json file with a SERVICE_API_URL key that's outside of version control for now. Hope that works with the distro.